Frontend/voitheia-App/src/app/components/request/request-form/request-form.component.scss

```diff
@@ -5,9 +5,32 @@
     display: flex;
     flex-direction: column;
     margin: 15px 0;
+    color: $gray-2;
+  }
+
+  .btn {
+    background-color: $brand-color;
+    color: $white;
+    border: none;
+    font-size: $normal;
```

Comment:
Should use Material buttons and designing should not take place in a specific component. We would like to have the same design for the whole page.

Reply:
obsolete code removed